src/test/java/uk/gov/pay/api/utils/ConnectorMockClient.java

```diff
@@ -60,11 +63,14 @@ private String createChargeResponse(long amount, String chargeId, String status,
                 .add("return_url", returnUrl)
                 .add("payment_provider", paymentProvider)
                 .add("created_date", createdDate)
-                .add("links", asList(links))
-                .build();
+                .add("links", asList(links));
+        if (gatewayTransactionId != null) {
+            jsonStringBuilder.add("gateway_transaction_id", gatewayTransactionId);
+        }
```

Comment:
This is not something we map into the Payment object of the public-api, do we really want to leak connector concerpt of gateway_transaction_id to the public-api?